### ISSUE TITLE
MultivariateNormalDistributionLoss has been removed

### ISSUE THREAD

**Issue by why362** [jdb78/pytorch-forecasting#1074]
- PyTorch-Forecasting version:0.10.1

I was trying to build a DeepAR model to solve a multivariate time series forcasting problem according to the tutorial in the offical website, https://pytorch-forecasting.readthedocs.io/en/latest/tutorials/deepar.html, only to find that the MultivariateNormalDistributionLoss had been removed from the source code. Why remove it? And what should I do to tackle this problem using DeepAR?

**Comment by erykml** [comment]
I did not dive that deep into the issue, but I also encountered it while using Google Colab.
Then, I installed the library locally, and the issue was gone.
Also, you can see here that the loss functions are not actually gone from the codebase: https://github.com/jdb78/pytorch-forecasting/blob/master/pytorch_forecasting/metrics/distributions.py#L43.

**Comment by RenaissanceEngineering** [comment]
Still an issue has anyone resolved this - on Google Colab?
